search/query_language_whitebox_test.go

```diff
@@ -94,6 +150,34 @@ func TestParseMap(t *testing.T) {
 		assert.Equal(t, expected, q)
 	})
 
+	t.Run("minimal OR and AND and Negate operation with EQ", func(t *testing.T) {
+		t.Parallel()
+		input := `
+		{"` + Q_OR + `": [{"` + Q_AND + `": [{"space": "openshiftio"},
```

Comment:
I am also not happy about this because `golint` is complaining. But it came from another PR.